topics/javafx3d/hello3d/build.gradle

```diff
@@ -1,31 +1,45 @@
 apply plugin: 'java'
 apply plugin: 'application'
 repositories {
-    maven {
-        url 'http://oss.sonatype.org/content/repositories/snapshots/'
-    }
+    mavenCentral()
 }
 mainClassName='org.projavafx.javafx3d.hello3d.Shapes'
+
+def currentOS = org.gradle.internal.os.OperatingSystem.current();
+def classifier
+if (currentOS.isWindows()) {
+    classifier = 'win'
+} else if (currentOS.isLinux()) {
+    classifier = 'linux'
+} else if (currentOS.isMacOsX()) {
+    classifier = 'mac'
+}
+
+
 compileJava {
     doFirst {
-        println classpath.asPath
+//        println classpath.asPath
         options.compilerArgs = [
             '--module-path', classpath.asPath,
+            '--add-modules', 'javafx.controls',
+            '--add-modules', 'javafx.graphics'
```

Comment:
this should not be needed. Once we're in the Java layer, the modules are fine (i.e. controls requires graphics)